src/alexandra_ai_eval/evaluator.py

```diff
@@ -325,6 +328,12 @@ def _send_results_to_leaderboard(self) -> List[bool]:
                 A list of booleans indicating whether the results were
                 successfully sent to the leaderboard.
         """
+        assert (
+            self.leaderboard_client is not None
+        ), "Leaderboard client is not initialized. Ensure that self.send_results_to_leaderboard is set to True."
```

Comment:
Maybe mention that the class needs to be re-initialised. I.e., it's not enough to just set `self.send_results_to_leaderboard` to True.